docs/references/observability.rst

```diff
@@ -362,6 +362,25 @@ For example, to only allow requests from an IP address to get the execution plan
  -- set this function on your postgrest.conf
  -- db-pre-request = filter_plan_requests
 
+.. _content-length_header:
+
+Content-Length header
+---------------------
+
+You can verify the response body size in bytes in the `Content-Length header <https://httpwg.org/specs/rfc9110.html#field.content-length>`_.
+
+.. code-block:: bash
+
+  curl -i 'localhost:3000/users'
+
+.. code-block:: http
+
+  HTTP/1.1 200 OK
+  Content-Length: 104
+
+Note that this header won't be returned on ``HEAD`` requests for optimization purposes (see :ref:`head_req`).
+The body size is also present in the :ref:`logs <pgrst_logging>` if they are enabled.
```

Comment:
`enabled` sounds like they need a config toggle, we use this term in other places with that meaning. Can we paraphrase?

Reply:
I modified to mention it directly "in the PostgREST logs", clicking the link should give the necessary info.